environment/env.go

```diff
@@ -60,7 +60,7 @@ const (
 	// ESVersion is the ElasticSearch version
 	ESVersion = "ES_VERSION"
 	// ESDefaultVersion is the default version
-	ESDefaultVersion = "v6"
+	ESDefaultVersion = "v7"
```

Comment:
Why do we need to update this version? This may break some developers' dev environment?

Reply:
I thought this was an unintentional default as Cadence server uses v7 since that's the running ES version but integration tests use v6. I can revert this to v6 if it's useful

Reply:
Our internal ES cluster is using v6, so I'd say let's revert it to v6?